lib/seed/Seeder.js

```diff
@@ -83,6 +84,11 @@ class Seeder {
 
   _getNewStubFileName(name) {
     if (name[0] === '-') name = name.slice(1);
+
+    if (this.config.timestampFilenamePrefix === true) {
+      name = yyyymmddhhmmss() + '_' + name;
```

Comment:
It would be preferable to use string template

Reply:
Yes, I was going to use string template, but I decided to just keep it the way it was.